libr/util/p_format.c

```diff
@@ -1388,10 +1388,19 @@ int r_print_format_struct_size(const char *f, RPrint *p, int mode, int n) {
 			break;
 		case '?':
 			{
+			const char *wordAtIndex = "";
+			if (idx < words) {
+				eprintf ("Index out of bounds\n");
+			} else {
+				wordAtIndex = r_str_word_get0 (args, idx);
+			}
 			const char *format = NULL;
```

Comment:
I know it's not a hard requirement, but I'd move these variable declarations at the beginning of the block, just before `const char *wordAtIndex`.